vsts-macos-ci.yml

```diff
@@ -0,0 +1,82 @@
+name: PR-$(System.PullRequest.PullRequestNumber)-$(Date:yyyyMMdd)$(Rev:.rr)
```

Comment:
It's better to move this file in a folder, like `vsts-ci`, so we don't have more and more files under the repository root.

Reply:
moved to a folder `.vsts-ci`